lua/entities/gmod_wire_cpu.lua

```diff
@@ -4,18 +4,17 @@ ENT.PrintName       = "Wire ZCPU"
 ENT.Author          = "Black Phoenix"
 ENT.WireDebugName	= "ZCPU"
 
-CreateConVar("wire_cpu_max_frequency", "1400000", FCVAR_REPLICATED)
+
 
 if CLIENT then return end -- No more client
 
 local cpu_max_frequency = 1400000
+CreateConVar("wire_cpu_max_frequency", "1400000", FCVAR_REPLICATED)
```

Comment:
And now as I already said in my previous review, instead of `"1400000"` use `cpu_max_frequency` as the second argument.